sse_test_subscriber.html

```diff
@@ -6,7 +6,7 @@
 <body>
     <script>
         if(typeof(EventSource) !== "undefined") {
-           var source = new EventSource('http://localhost:8080/user/skrzepto');
+           var source = new EventSource('http://76.10.221.44:8080/user/skrzepto');
```

Comment:
if the goal of this file is to help test a dev version, I doubt we want to point to your IP :)